activerecord/test/cases/dirty_test.rb

```diff
@@ -942,16 +942,17 @@ def check_around
   end
 
   test "partial insert off with unchanged default function attribute" do
+    freeze_time
     with_partial_writes Aircraft, false do
       aircraft = Aircraft.new(name: "Boeing")
       assert_equal "Boeing", aircraft.name
 
       aircraft.save!
-      expected_manufactured_at = Time.now
+      expected_manufactured_at = Time.current
       aircraft.reload
 
       assert_equal "Boeing", aircraft.name
-      assert_in_delta expected_manufactured_at, aircraft.manufactured_at, 1
+      assert_equal expected_manufactured_at.to_i, aircraft.manufactured_at.to_i
```

Comment:
Can you update this line to match the assertion in `partial insert off with changed default function attribute`, so that we're using parallel constructions in these two related tests?